src/main/webapp/app/exam/participate/exercises/file-upload/file-upload-exam-submission.component.ts

```diff
@@ -166,4 +167,10 @@ export class FileUploadExamSubmissionComponent extends ExamSubmissionComponent i
     private onError() {
         this.alertService.error(this.translateService.instant('error.fileUploadSavingError'));
     }
+
+    setSubmissionVersion(submissionVersion: SubmissionVersion): void {
+        // submission versions are not supported for file upload exercises
+        this.submissionVersion = submissionVersion;
```

Comment:
```suggestion
```
Is this needed?

Reply:
yes, because otherwise, es-lint fails either because it's unused or if I use ts-ignore es-lint fails because ts-ignore is used.

Reply:
Can you add a comment to explain that?